src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -2607,21 +2793,19 @@ class Animal {
 }
 ```
 
+
+
 In JavaScript, this is a convention,
 but in Dart this is a full language feature
-enforced by the compiler. 
-
-A difference that should be noted is that in Dart,
-private members are not private to the class,
-but are private to the library,
-meaning that private members can still be accessed
-from code that is considered part of the same library.
-By default, this is possible anywhere in the same file,
-so private class members can still be accessed from code
-in the same file. While you can expand this library scope
-beyond a single file using the `part` directive,
-it's generally advised to [avoid doing so][] and is usually
-reserved for code generators. 
+enforced by the compiler.
+
+Dart makes private members private to the library, not the class.
+This means that you can access private members from code in the same library.
+By default, this works anywhere in the same file.
+Private class members can be accessed from code in the same file.
+Using the `part` directive to expand the scope of a library scope a single file.
+When possible, [avoid doing so using `part`][]. The common practice would be
```

Comment:
Just ``[avoid using `part`]``. There are other reasons to use `part` files and you should avoid those too if you can.